src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -166,7 +165,7 @@ def _update_expiration_warnings(self, loop_start_time):
                 'subject': self.config['default_subject'],
                 'body': 'The expiration status of one or more keys have changed.'}
             encrypted_message = self._build_encrypted_message(message_dict, loop_start_time)
-            self.mailsender.sendmail(encrypted_message, self.valid_recipient_emails)
+            self._sendmail(encrypted_message, self.valid_recipient_emails)
```

Comment:
Oh, this is an actual error. You need 'self._sendmail(message_object=encrypted_message, self.valid_recipient_emails)' instead of 'self._sendmail(encrypted_message, self.valid_recipient_emails)'.